Xamarin.PropertyEditing.Windows/NumericUpDownControl.cs

```diff
@@ -62,6 +62,11 @@ internal abstract class NumericUpDownControl<T>
 		: Control
 		where T : struct, IComparable<T>
 	{
+		static NumericUpDownControl ()
+		{
+			FocusableProperty.OverrideMetadata (typeof (NumericUpDownControl<T>), new FrameworkPropertyMetadata (false));
```

Comment:
@alanmcgovern Ah I didn't look close enough, I thought the NumericUpDown was both the input box and up/down controls, hence why I thought it needed focus.